### DISCUSSION TITLE
Hardhat chain 1337 with Metamask - can't import tokens

### DISCUSSION THREAD

**Discussion by rick897887** [NomicFoundation/hardhat#3539] [Troubleshooting]
I'm using a chain ID of 1337 with Hardhat. When I try to import a custom ERC20 deployed on my local Hardhat node into Metamask, I get a "TokenService Error: ChainId 0x539 is not supported" error message. Is anyone else getting this? A Google search for this exact error message reveals two results only.

**Comment by fvictorio** [comment]
@rick897887 try changing the chain id of the localhost network to 31337. To do that go to Metamask settings -> Networks -> Select "Localhost 8545" and change the "Chain ID" field.

I have that configuration and I was able to import a token I deployed to a local node.

**Reply by rick897887** [reply]
Yeah that works, seems Metamask has fixed the problem with chain 31337 and now it works fine, thanks!
